script/Arcueid/Day 7/07_00_ARC07_1.txt

```diff
@@ -148,7 +148,7 @@ We'll never see each other again.
 -- Page 12, Offset 16593.
 -- あんな物騒な女と関わるなんて、金輪際あってはいけない。
 // 金輪際 translated to similar idiom
-You couldn't convince me to get involved with that walking disaster of a woman again,
+You couldn't convince me to get involved with that walking calamity of a woman again,
```

Comment:
Walking disaster is a bit of a set phrase, I might keep 
![2022-01-21-211051_1292x463_scrot](https://user-images.githubusercontent.com/2476799/150620859-fcb44901-51b6-4aa6-aa59-54ad6f728f0d.png)

Reply:
I originally wanted to avoid 'disaster' because of the fact that it is a set phrase. It more implies that the target's life is a mess than anything else. Calamity means the same thing but is free from the connotaion. However I don't know how it was originally intended so I'll revert it.

Reply:
Hrm, true I forgot that it's more like their a klutz than an animate disaster.

Reply:
JP here is just 物騒な女, 'dangerous woman', so if you can come up with something cleaner have at it